src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -161,8 +161,9 @@ class CosmasGoogleCloudService(private val bucketName: String,
     override fun getVersion(request: CosmasProto.GetVersionRequest,
                             responseObserver: StreamObserver<CosmasProto.GetVersionResponse>) {
         LOG.info("Get request for version ${request.version} file # ${request.fileId}")
+        val version = if (request.version == -1L) null else request.version
```

Comment:
This means "gimme the last version"? Can you please add a comment to the version field in proto file and here about this special meaning of -1 and the semantics of other values of "version"? I mean, they are supposed to be "generation" values, right? If yes then I  suggest renaming: we already have "generation" in the protobuffers, so let's call the same things with the same names.

Reply:
done